### PR TITLE
Byebye [!=]= 1

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

**Comment by wbhart** [comment]
Hmm looks ok, but tests seem to say otherwise.

**Comment by wbhart** [comment]
Ah yeah, just missing parenthesis in PuiseuxSeries.jl.

File: src/generic/PuiseuxSeries.jl [hunk 1]
```diff
@@ -532,7 +532,7 @@ function ^(a::PuiseuxSeriesElem{T}, b::Int) where T <: RingElement
 end
 
 function ^(a::PuiseuxSeriesElem{T}, b::Rational{Int}) where T <: RingElement
-   (pol_length(a.data) != 1 || polcoeff(a.data, 0) != 1) && error("Rational power not implemented")
+   (pol_length(a.data) != 1 || !isone(polcoeff(a.data, 0)) && error("Rational power not implemented")
```
Comment (thofma) [inline src/generic/PuiseuxSeries.jl:535]:
```suggestion
   (pol_length(a.data) != 1 || !isone(polcoeff(a.data, 0))) && error("Rational power not implemented")
```

**Review by wbhart — APPROVED** [review]

**Review by tthsqe12 — COMMENTED** [review]
I cannot condone the abuses of the type system that would necessitate such changes.

**Pull request merged**